string cmd

---
string result
uint32 error_code
